haskell/test/Test/Ligo/TreasuryDAO.hs

```diff
@@ -213,7 +198,7 @@ tokenTransferType contractAddr fromAddr toAddr = Token_transfer_type TokenTransf
       { tiFrom = fromAddr
       , tiTxs = [ FA2.TransferDestination
           { tdTo = toAddr
-          , tdTokenId = unfrozenTokenId
+          , tdTokenId = frozenTokenId
```

Comment:
For the same reason as in `RegistryDAO` I think this should be changed from to be `FA2.TokenId 0`.

Actually even better I think it's more clear if we use `FA2.theTokenId` in both places instead.

Reply:
I have change this where possible.